tests/network/vsftpd.pm

```diff
@@ -33,34 +33,25 @@ use utils 'zypper_call';
 sub run {
     my $password = $testapi::password;
     select_console 'root-console';
-    if (check_var('HOSTNAME', 'server')) {
-        barrier_create('SSHD_READY', 2);
-        zypper_call 'in vsftpd';
-    }
-    else {
-        zypper_call 'in expect';
-    }
+    zypper_call 'in vsftpd';
+    zypper_call 'in expect';
```

Comment:
Why not both in single zypper_call ?

Reply:
fixed

Reply:
> It could run as single machine test, I will change it in our tests.

Awesome! Thanks, as this affects you more than us, lmk when it's ready on your side, so that we can merge this then :)

Reply:
btw, don't you want to move this test into console ? not sure where you want to schedule the test

Reply:
done